frontend/src/components/WtGIContent.tsx

```diff
@@ -8,25 +8,23 @@ const WtGIContent = () => {
 
   return (
     <GridContainer
-      className="desktop:padding-y-4 tablet:padding-y-2 padding-y-1"
+      className="padding-y-1 tablet:padding-y-3 desktop-lg:padding-y-6"
```

Comment:
Again just reversing the order to be mobile-first and tweaking the whitespace here.